Classes/Service/Tika/ServerService.php

```diff
@@ -120,9 +120,8 @@ public function ping()
     public function extractMetadataFromLocalFile($fileName)
     {
         $content = $this->send('PUT', '/meta', 'application/json', $fileName);
-        $metadata = json_decode($content, true);
-
-        return $metadata ?? [];
+        $metadata = empty($content) ? [] : json_decode($content, true);
+        return $metadata;
```

Comment:
IMHO we still need the `return $metadata ?? []` in case `json_decode` fails somehow

Reply:
but the crash occurs in 123... the execution stops before the ternary can be called... 

PHP 8.2.15

Reply:
I didn't say the rest of the patch didn't look good :)

Reply:
I mean that if somehow `$content` is not empty and CANNOT be properly decoded as JSON, then the second part of the ternary with the `json_decode` you use will put something that is not an array into `$metadata`, thus the return statement should (still) take care of that imho.

Reply:
the function json_decode() can't use NULL as first parameter... at least PHP 8.2.15.. so if $content is NULL in 123, the execution stops... no matter what happens afterwards...